apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerUtils.java

```diff
@@ -3264,6 +3264,12 @@ public static void logBeneIdToMdc(Collection<String> beneIds) {
     }
   }
 
+  public static void logMbiHashToMdc(String mbiHash) {
```

Comment:
We need to move this to https://github.com/CMSgov/beneficiary-fhir-data/blob/master/apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/TransformerUtilsV2.java#L1603

Reply:
I have moved the method.